Source/Blazorise/Enums/Enumeration.cs

```diff
@@ -55,12 +55,16 @@ protected Enumeration( T parent, string name )
             return new Enumeration<T>( name );
         }
 
+        /// <summary>
+        /// Build an enumeration while traversing up to the parent.
+        /// </summary>
+        /// <returns></returns>
         private string BuildName()
         {
             var sb = new StringBuilder();
 
-            if ( Parent != null )
-                sb.Append( Parent.Name ).Append( ' ' );
+            if ( ParentEnumeration != null )
+                sb.Append( ParentEnumeration.Name ).Append( ' ' );
```

Comment:
You mean like this `if ( ParentEnumeration != default )`?

Reply:
Yes because in the constructor you set it to default. However I was under the wrong impression that you could override the default.... Apparently not... So it probably does not matter much, only if a value type needed to be correctly Considered != default 
https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/default

So basically nvm my suggestion haha